src/lib/datasource/FileReadWriteDatasource.js

```diff
@@ -55,6 +58,47 @@ class FileReadWriteDatasource extends FileReadDatasource {
         });
     }
 
+    /**
+     * 
+     * @param {*} keysArray 
+     */
+    mget(keysArray) {
+        // todo: set some check if args is not an array
+       /* if(typeof keysArray !== 'Array') {
+            return Promise.reject(new Error('Error, parameter should be an array of ids'));
+        }*/   
+        return this.readFile().then((fileData)=> {
+            return keysArray.map((key) => fileData[key])
+                            .map(undefinedToNull);
```

Comment:
this is bad practice for indentation.
U'll have to add more spaces in case of variable rename for egample.
Should be like this:
```
return object
    .method()
    .method()
    .method();
```

Reply:
Sometimes, I fucking my brain with indentation. And looking for the case, how it can be more readable.